projects/packages/videopress/src/client/state/utils/map-videos.ts

```diff
@@ -66,10 +66,10 @@ export const mapVideoFromWPV2MediaEndpoint = (
 		rating,
 		privacySetting,
 		needsPlaybackToken,
+		width,
+		height,
```

Comment:
This info isn't being used in other places? I remembered it's used, but maybe I'm wrong

Reply:
I did not find any place that uses `width` or `height` from `poster`.